packages/appium/test/fixtures/test-driver/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "peerDependencies": {
-    "appium": "^2.0.0-beta.42"
+    "appium": "^2.0.0-beta.42 || ^3.0.0-beta.0"
```

Comment:
I'd say for fake/test drivers it makes sense to only make them compatible with ^3. We must do it anyway later after we bump the base driver